translations/catroweb.en.yml

```diff
@@ -184,6 +184,9 @@ programs:
   link: Copy project link to clipboard
   reactionsText: Reactions
   downloadErrorText: Error occurred while downloading the project. Please try again later.
+  showTranslation: Show Translation
+  hideTranslation: Hide Translation
+  translatedByText: Translated by %provider% from %sourceLanguage% to %targetLanguage%
```

Comment:
i thought of a better name for this `translationByline`, but it's not that important

Reply:
Updated variable name